### PR TITLE
Align texts messages in the ubiquity installer scripts.

### PR DESCRIPTION
**Pull request by shay-berman**
According to Dima review.

**Here is full installation flow detail output step by step:**
```
[root@xavi-k8s-master installer-for-ibm-storage-enabler-for-containers]#  ./ubiquity_installer.sh -s update-ymls -c  /var/tmp/new/ubiquity_installer.conf
Executing STEP [update-ymls]...
Updating yml files with placeholders from /var/tmp/new/ubiquity_installer.conf file. Are you sure (y/n): y
Update placeholder [UBIQUITY_IMAGE                ] in files : ./yamls/ubiquity-deployment.yml  
Update placeholder [UBIQUITY_DB_IMAGE             ] in files : ./yamls/ubiquity-db-deployment.yml  
Update placeholder [UBIQUITY_K8S_PROVISIONER_IMAGE] in files : ./yamls/ubiquity-k8s-provisioner-deployment.yml  
Update placeholder [UBIQUITY_K8S_FLEX_IMAGE       ] in files : ./yamls/ubiquity-k8s-flex-daemonset.yml  
Update placeholder [SCBE_MANAGEMENT_IP_VALUE      ] in files : ./ubiquity-configmap.yml  
Update placeholder [SCBE_MANAGEMENT_PORT_VALUE    ] in files : ./ubiquity-configmap.yml  
Update placeholder [SCBE_DEFAULT_SERVICE_VALUE    ] in files : ./ubiquity-configmap.yml  
Update placeholder [UBIQUITY_INSTANCE_NAME_VALUE  ] in files : ./ubiquity-configmap.yml  
Update placeholder [DEFAULT_FSTYPE_VALUE          ] in files : ./ubiquity-configmap.yml  
Update placeholder [DEFAULT_VOLUME_SIZE_VALUE     ] in files : ./ubiquity-configmap.yml  
Update placeholder [SKIP_RESCAN_ISCSI_VALUE       ] in files : ./ubiquity-configmap.yml  
Update placeholder [LOG_LEVEL_VALUE               ] in files : ./ubiquity-configmap.yml  
Update placeholder [SSL_MODE_VALUE                ] in files : ./ubiquity-configmap.yml  
Update (base64) placeholder [SCBE_USERNAME_VALUE           ] in files : ./scbe-credentials-secret.yml  
Update (base64) placeholder [SCBE_PASSWORD_VALUE           ] in files : ./scbe-credentials-secret.yml  
Update (base64) placeholder [UBIQUITY_DB_USERNAME_VALUE    ] in files : ./ubiquity-db-credentials-secret.yml  
Update (base64) placeholder [UBIQUITY_DB_PASSWORD_VALUE    ] in files : ./ubiquity-db-credentials-secret.yml  
Update placeholder [STORAGE_CLASS_NAME_VALUE      ] in files : ./yamls/storage-class.yml ./yamls/ubiquity-db-pvc.yml ./yamls/sanity_yamls/sanity-pvc.yml  
Update placeholder [STORAGE_CLASS_PROFILE_VALUE   ] in files : ./yamls/storage-class.yml  
Update placeholder [STORAGE_CLASS_FSTYPE_VALUE    ] in files : ./yamls/storage-class.yml  
Finished updating yml files according to /var/tmp/new/ubiquity_installer.conf





[root@xavi-k8s-master installer-for-ibm-storage-enabler-for-containers]# ./ubiquity_installer.sh -s install -k ~/.kube/config 
Executing STEP [install]...
Starting installation  "IBM Storage Enabler for Containers"...
Installing on the namespace  [ubiquity].
namespace "ubiquity" created
service "ubiquity" created
service "ubiquity-db" created
Update the UBIQUITY-IP-ADDRESS: <IP> in the file [./yamls/../ubiquity-configmap.yml]
configmap "ubiquity-configmap" created
secret "scbe-credentials" created
secret "ubiquity-db-credentials" created
deployment "ubiquity" created
Waiting for deployment [ubiquity] to be in Running state...
deployment [ubiquity] reached the expected generation [1]
deployment [ubiquity] available replicas are [0] while expected value is [1], sleeping [5 sec] before retrying to check [0/20]
deployment [ubiquity] available replicas are [0] while expected value is [1], sleeping [5 sec] before retrying to check [1/20]
deployment [ubiquity] reached the expected replicas [1]
Creating k8s-config for ubiquity-k8s-provisioner from file [/root/.kube/config].
configmap "k8s-config" created
deployment "ubiquity-k8s-provisioner" created
Waiting for deployment [ubiquity-k8s-provisioner] to be in Running state...
deployment [ubiquity-k8s-provisioner] reached the expected generation [1]
deployment [ubiquity-k8s-provisioner] available replicas are [0] while expected value is [1], sleeping [5 sec] before retrying to check [0/20]
deployment [ubiquity-k8s-provisioner] available replicas are [0] while expected value is [1], sleeping [5 sec] before retrying to check [1/20]
deployment [ubiquity-k8s-provisioner] reached the expected replicas [1]
storageclass "gold" created
persistentvolumeclaim "ibm-ubiquity-db" created
Waiting for ibm-ubiquity-db PVC to be created
pvc [ibm-ubiquity-db] status is [Pending] while expected status is [Bound]. sleeping [5 sec] before retrying to check [0/60]
pvc [ibm-ubiquity-db] status is [Pending] while expected status is [Bound]. sleeping [5 sec] before retrying to check [1/60]
pvc [ibm-ubiquity-db] status [Bound] as expected (after 2/60 tries)
Waiting for ibm-ubiquity-db PV to be created
pv [ibm-ubiquity-db] status [Bound] as expected (after 0/20 tries)
Deploying FlexVolume driver as a daemonset on all nodes and masters. The daemonset will use the ubiquity service IP.
daemonset "ubiquity-k8s-flex" created
Waiting for daemonset [ubiquity-k8s-flex] to be in Running state...
daemonset [ubiquity-k8s-flex] reached the expected generation [1]
daemonset [ubiquity-k8s-flex] available pods are [0] while expected quantity is [3], sleeping [3 sec] before retrying to check [0/20]
daemonset [ubiquity-k8s-flex] available pods are [0] while expected quantity is [3], sleeping [3 sec] before retrying to check [1/20]
daemonset [ubiquity-k8s-flex] reached the expected available pods [3]

"IBM Storage Enabler for Containers" Installation finished, but the deployment is not ready yet.
  Perform the following: 
     (1) Manually restart the kubelet service on all Kubernetes nodes to reload the new FlexVolume driver.
     (2) Deploy ubiquity-db by $> ./ubiquity_installer.sh -s create-ubiquity-db -n ubiquity
     Note : View status by $> ./ubiquity_cli.sh -a status -n ubiquity





[root@xavi-k8s-master installer-for-ibm-storage-enabler-for-containers]#  ./ubiquity_installer.sh -s create-ubiquity-db -n ubiquity
Executing STEP [create-ubiquity-db]...
Creating ubiquity-db deployment... (Assuming that the IBM Storage Kubernetes FlexVolume(ubiquity-k8s-flex) plugin is already loaded on all the nodes)
deployment "ubiquity-db" created
Waiting for deployment [ubiquity-db] to be created...
Waiting for deployment [ubiquity-db] to be in Running state...
deployment [ubiquity-db] reached the expected generation [1]
deployment [ubiquity-db] available replicas are [0] while expected value is [1], sleeping [5 sec] before retrying to check [0/50]
...
...
deployment [ubiquity-db] available replicas are [0] while expected value is [1], sleeping [5 sec] before retrying to check [6/50]
deployment [ubiquity-db] reached the expected replicas [1]

"IBM Storage Enabler for Containers" installation finished successfully in the Kubernetes cluster. 
           - Get status      $> ./ubiquity_cli.sh -a status -n ubiquity
           - Run sanity test $> ./ubiquity_cli.sh -a sanity -n ubiquity





[root@xavi-k8s-master installer-for-ibm-storage-enabler-for-containers]# ./ubiquity_cli.sh -a status -n ubiquity
Working in namespace [ubiquity].
kubectl get storageclass | egrep "ubiquity|^NAME"
---------------------------------------------------------------------
NAME      PROVISIONER
gold      ubiquity/flex

kubectl get --namespace ubiquity secret/ubiquity-db-credentials secret/scbe-credentials cm/k8s-config cm/ubiquity-configmap pv/ibm-ubiquity-db pvc/ibm-ubiquity-db svc/ubiquity svc/ubiquity-db daemonset/ubiquity-k8s-flex deploy/ubiquity deploy/ubiquity-db deploy/ubiquity-k8s-provisioner
---------------------------------------------------------------------
NAME                              TYPE      DATA      AGE
secrets/ubiquity-db-credentials   Opaque    3         16m
secrets/scbe-credentials          Opaque    2         16m

NAME                    DATA      AGE
cm/k8s-config           1         16m
cm/ubiquity-configmap   10        16m

NAME                 CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS    CLAIM                      STORAGECLASS   REASON    AGE
pv/ibm-ubiquity-db   20Gi       RWO            Delete           Bound     ubiquity/ibm-ubiquity-db   gold                     16m

NAME                  STATUS    VOLUME            CAPACITY   ACCESS MODES   STORAGECLASS   AGE
pvc/ibm-ubiquity-db   Bound     ibm-ubiquity-db   20Gi       RWO            gold           16m

NAME              TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
svc/ubiquity      ClusterIP   <IP>   <none>        9999/TCP   16m
svc/ubiquity-db   ClusterIP  <IP>   <none>        5432/TCP   16m

NAME                   DESIRED   CURRENT   READY     UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
ds/ubiquity-k8s-flex   3         3         3         3            3           <none>          16m

NAME                              DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
deploy/ubiquity                   1         1         1            1           16m
deploy/ubiquity-db                1         1         1            1           15m
deploy/ubiquity-k8s-provisioner   1         1         1            1           16m

kubectl get --namespace ubiquity pod | egrep "^ubiquity|^NAME"
---------------------------------------------------------------------
NAME                                        READY     STATUS    RESTARTS   AGE
ubiquity-9785f567f-8bddb                    1/1       Running   0          16m
ubiquity-db-bc467d898-p672h                 1/1       Running   0          15m
ubiquity-k8s-flex-7scfc                     1/1       Running   0          16m
ubiquity-k8s-flex-9f7vc                     1/1       Running   0          16m
ubiquity-k8s-flex-qc4nh                     1/1       Running   0          16m
ubiquity-k8s-provisioner-74774d65f5-svxgv   1/1       Running   0          16m







[root@xavi-k8s-master installer-for-ibm-storage-enabler-for-containers]# ./ubiquity_cli.sh -a sanity -n ubiquity
Working in namespace [ubiquity].
--------------------------------------------------------------
Sanity description:
    1. Create sanity-pvc, sanity-pod and wait for creation.
    2. Delete the sanity-pod, sanity-pvc and wait for deletion.
    Note: Uses yml files from directory ./yamls/sanity_yamls
--------------------------------------------------------------

persistentvolumeclaim "sanity-pvc" created
pvc [sanity-pvc] status is [Pending] while expected status is [Bound]. sleeping [3 sec] before retrying to check [0/10]
pvc [sanity-pvc] status [Bound] as expected (after 1/10 tries)
pod "sanity-pod" created
pod [sanity-pod] status is [Pending] while expected status is [Running]. sleeping [3 sec] before retrying to check [0/100]
pod [sanity-pod] status is [Pending] while expected status is [Running]. sleeping [3 sec] before retrying to check [1/100]
pod [sanity-pod] status is [Pending] while expected status is [Running]. sleeping [3 sec] before retrying to check [2/100]
pod [sanity-pod] status is [Pending] while expected status is [Running]. sleeping [3 sec] before retrying to check [3/100]
pod [sanity-pod] status [Running] as expected (after 4/100 tries)
pod "sanity-pod" deleted
NAME         READY     STATUS        RESTARTS   AGE
sanity-pod   1/1       Terminating   0          13s
pod [sanity-pod] still exists. sleeping [3 sec] before retrying to check [0/100]
NAME         READY     STATUS        RESTARTS   AGE
sanity-pod   1/1       Terminating   0          16s
...
...
pod [sanity-pod] still exists. sleeping [3 sec] before retrying to check [14/100]
pod [sanity-pod] was deleted (after 15/100 tries)
persistentvolumeclaim "sanity-pvc" deleted
pvc [sanity-pvc] was deleted (after 0/10 tries)
NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS     CLAIM                 STORAGECLASS   REASON    AGE
pvc-449ede89-d54a-11e7-9359-005056a45c0b   1Gi        RWO            Delete           Released   ubiquity/sanity-pvc   gold                     1m
pv [pvc-449ede89-d54a-11e7-9359-005056a45c0b] still exists. sleeping [2 sec] before retrying to check [0/10]
pv [pvc-449ede89-d54a-11e7-9359-005056a45c0b] was deleted (after 1/10 tries)

"IBM Storage Enabler for Containers" sanity finished successfully.
```

**Here is uninstall**
```
[root@xavi-k8s-master installer-for-ibm-storage-enabler-for-containers]# ./ubiquity_uninstall.sh 
Attention: Uninstall "IBM Storage Enabler for Containers" will delete all Ubiquity components, including ubiquity-db, credentials and namespace.
Are you sure (y/n): y
Start to uninstall "IBM Storage Enabler for Containers" from namespace [ubiquity]...
deployment "ubiquity-db" deleted
Wait for ubiquity-db deployment deletion...
deployment [ubiquity-db] was deleted (after 0/10 tries)
ubiquity-db-bc467d898-p672h                 0/1       Terminating   0          21m
pod [ubiquity-db-] still exists. sleeping [4 sec] before retrying to check [0/10]
....
....
pod [ubiquity-db-] still exists. sleeping [4 sec] before retrying to check [4/10]
ubiquity-db-bc467d898-p672h                 0/1       Terminating   0          21m
pod [ubiquity-db-] still exists. sleeping [4 sec] before retrying to check [5/10]
pod [ubiquity-db-] was deleted (after 6/10 tries)
persistentvolumeclaim "ibm-ubiquity-db" deleted
Waiting for PVC ibm-ubiquity-db and PV ibm-ubiquity-db to be deleted, before deleting Ubiquity and Provisioner.
pvc [ibm-ubiquity-db] was deleted (after 0/10 tries)
NAME              CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS     CLAIM                      STORAGECLASS   REASON    AGE
ibm-ubiquity-db   20Gi       RWO            Delete           Released   ubiquity/ibm-ubiquity-db   gold                     22m
pv [ibm-ubiquity-db] still exists. sleeping [3 sec] before retrying to check [0/10]
NAME              CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS     CLAIM                      STORAGECLASS   REASON    AGE
ibm-ubiquity-db   20Gi       RWO            Delete           Released   ubiquity/ibm-ubiquity-db   gold                     22m
pv [ibm-ubiquity-db] still exists. sleeping [3 sec] before retrying to check [1/10]
NAME              CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS     CLAIM                      STORAGECLASS   REASON    AGE
ibm-ubiquity-db   20Gi       RWO            Delete           Released   ubiquity/ibm-ubiquity-db   gold                     22m
pv [ibm-ubiquity-db] still exists. sleeping [3 sec] before retrying to check [2/10]
pv [ibm-ubiquity-db] was deleted (after 3/10 tries)
storageclass "gold" deleted
deployment "ubiquity-k8s-provisioner" deleted
configmap "k8s-config" deleted
daemonset "ubiquity-k8s-flex" deleted
deployment "ubiquity" deleted
configmap "ubiquity-configmap" deleted
secret "scbe-credentials" deleted
secret "ubiquity-db-credentials" deleted
service "ubiquity" deleted
service "ubiquity-db" deleted
namespace "ubiquity" deleted

"IBM Storage Enabler for Containers" uninstall finished.
```

---
This change is [Reviewable](https://reviewable.io/reviews/ibm/ubiquity-k8s/147)

### CODE REVIEW

File: scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh [hunk 1]
```diff
@@ -17,24 +17,11 @@
 #*******************************************************************************
 
 # -------------------------------------------------------------------------
-# "IBM Storage Enabler for Containers" cli tool.
-# The tool is an helper to do the following actions : start, stop, status, collect_logs, sanity.
-# The script assume that you already installed ubiquity via the ubiquity_install.sh script.
+# IBM Storage Enabler for Containers CLI tool
+# This is a utility for starting/stoping IBM Storage Enabler for Containers (Ubiquity), retrieving its status, collecting logs, running sanity test.
+# The script asumes that you already installed Ubiquity using the ubiquity_install.sh script.
```
Comment (alonm) [inline scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh]: 1. Should limit line length to 80 characters.
2. I suggest making a general statement and then giving the options:
   This utility enables various operations with IBM Storage Enabler...:
   Start/Stop the service
   Get Status
3. asumes => assumes

File: scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh [hunk 2]
```diff
@@ -73,28 +68,35 @@ function start()
...
     echo "Wait for ubiquity-db deployment deletion..."
     wait_for_item_to_delete deployment ubiquity-db 10 4 "" $NS
-    wait_for_item_to_delete pod "ubiquity-db-" 10 4 regex $NS  # to match the prefix of the pod
+    wait_for_item_to_delete pod "ubiquity-db-" 10 4 regex $NS  # to match the pod prefix
 
     $kubectl_delete -f $YML_DIR/${UBIQUITY_PROVISIONER_DEPLOY_YML}
     $kubectl_delete -f ${YML_DIR}/${UBIQUITY_FLEX_DAEMONSET_YML}
     $kubectl_delete -f $YML_DIR/${UBIQUITY_DEPLOY_YML}
-    echo "Finished to stop ubiquity deployments successfully.   Note : View ubiquity status by : $> $0 -a status -n $NS"
+    echo "Stopped Ubiquity components.   Note: View deployment status by: $> $0 -a status -n $NS"
```
Comment (alonm) [inline scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh]: Are the components fully stopped at this stage? Or is it an async operation?

File: scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh [hunk 3]
```diff
@@ -170,13 +172,13 @@ function status()
 
     if [ $rc != 0 ]; then
        echo ""
-       echo "Ubiquity status [NOT ok]. Some components are missing(review the output above)"
+       echo "Ubiquity status [not OK]. Some components are missing(review the output above)"
```
Comment (alonm) [inline scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh]: I think the convention is NOT OK (all capitals)

File: scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_lib.sh [hunk 4]
```diff
@@ -237,56 +237,56 @@ function wait_for_daemonset(){
     generation=$(get_generation daemonset $item_name $ns)
     while [[ $(get_observed_generation daemonset $item_name $ns) -lt ${generation} ]]; do
       if [ "$retries" -eq 0 ]; then
-          echo "Error: ${item_type} [${item_name}] generation $(get_observed_generation daemonset $item_name $ns) < ${generation}, even after all ${max_retries} retries. exit."
+          echo "Error: ${item_type} [${item_name}] generation is [$(get_observed_generation daemonset $item_name $ns]) while the expected value is [${generation}], after all ${max_retries} retries. Exiting."
           echo "$EXIT_WAIT_TIMEOUT_MESSAGE"
           exit 2
       else
-          echo "${item_type} [${item_name}] generation $(get_observed_generation daemonset $item_name $ns) < ${generation}, sleeping [${delay} sec] before retry to check [$(($max_retries - $retries))/${max_retries}] "
+          echo "${item_type} [${item_name}] generation is [$(get_observed_generation daemonset $item_name $ns)] while the expected value is [${generation}], sleeping [${delay} sec] before retrying to check [$(($max_retries - $retries))/${max_retries}] "
```
Comment (alonm) [inline scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_lib.sh]: better break the lines

File: scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_uninstall.sh [hunk 5]
```diff
@@ -17,16 +17,16 @@
...
 
 # -------------------------------------------------------------------------
-# "IBM Storage Enabler for Containers" uninstall script.
-# the following IBM components inside the kubernetes cluster:
-#       "IBM Storage Enabler for Containers"
-#       "IBM Storage Dynamic Provisioner for Kubernetes"
-#       "IBM Storage Flex Volume for Kubernetes"
-# The script uninstall all Ubiquity components (including the ubiquity data which locate in PV ibm-ubiquity-db).
+# IBM Storage Enabler for Containers uninstall script
+# The script unistalls following IBM Storage Enabler for Containers (Ubiquity) components from the Kubernetes cluster:
```
Comment (alonm) [inline scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_uninstall.sh]: issing 'n' in uninstalls

File: scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh [hunk 6]
```diff
@@ -17,24 +17,11 @@
 #*******************************************************************************
 
 # -------------------------------------------------------------------------
-# "IBM Storage Enabler for Containers" cli tool.
-# The tool is an helper to do the following actions : start, stop, status, collect_logs, sanity.
-# The script assume that you already installed ubiquity via the ubiquity_install.sh script.
+# IBM Storage Enabler for Containers CLI tool
+# This is a utility for starting/stoping IBM Storage Enabler for Containers (Ubiquity), retrieving its status, collecting logs, running sanity test.
```
Comment (alonm) [inline scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh]: should be 'stopping'

**Comment by shay-berman** [comment]
Review status: 0 of 4 files reviewed at latest revision, 6 unresolved discussions.

---

*[scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh, line 22 at r1](https://reviewable.io:443/reviews/ibm/ubiquity-k8s/147#-L-8F1P-tGK932i6ZBVA-r1-22:-L-8TNXl_6J01ejprib_:bwc2f8i) ([raw file](https://github.com/ibm/ubiquity-k8s/blob/f3ed1cef1494f9a90ea111c03b5c2d9e00a5b72c/scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh#L22)):*
> assumes
done (1 and 3)

---

*[scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh, line 94 at r1](https://reviewable.io:443/reviews/ibm/ubiquity-k8s/147#-L-8F1P-tGK932i6ZBVA-r1-94:-L-8UF7Ik9zI7mIIHAMD:b-173r2e) ([raw file](https://github.com/ibm/ubiquity-k8s/blob/f3ed1cef1494f9a90ea111c03b5c2d9e00a5b72c/scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh#L94)):*
<details><summary><i>Previously, alonm wrote…</i></summary><blockquote>

Are the components fully stopped at this stage? Or is it an async operation?
</blockquote></details>

async - this is why there is NOTE to view the deployment s status.
not going to add wait for deployments here (at least not now).

---

*[scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh, line 175 at r1](https://reviewable.io:443/reviews/ibm/ubiquity-k8s/147#-L-8F1P-tGK932i6ZBVA-r1-175:-L-8UVVGtjqCGaYVCbkO:bikw4ng) ([raw file](https://github.com/ibm/ubiquity-k8s/blob/f3ed1cef1494f9a90ea111c03b5c2d9e00a5b72c/scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_cli.sh#L175)):*
<details><summary><i>Previously, alonm wrote…</i></summary><blockquote>

I think the convention is NOT OK (all capitals)
</blockquote></details>

Follow Dima instructions.

---

*[scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_lib.sh, line 244 at r1](https://reviewable.io:443/reviews/ibm/ubiquity-k8s/147#-L-8F1P-tGK932i6ZBVC-r1-244:-L-8UfZpWuFJuHnG2kS5:b-epxkug) ([raw file](https://github.com/ibm/ubiquity-k8s/blob/f3ed1cef1494f9a90ea111c03b5c2d9e00a5b72c/scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_lib.sh#L244)):*
<details><summary><i>Previously, alonm wrote…</i></summary><blockquote>

better break the lines
</blockquote></details>

prefer not
its look OK as it echo out to the user.

---

*[scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_uninstall.sh, line 21 at r1](https://reviewable.io:443/reviews/ibm/ubiquity-k8s/147#-L-8F1P-tGK932i6ZBVD-r1-21:-L-8UszhBDvM_tme1gAU:bd30ifx) ([raw file](https://github.com/ibm/ubiquity-k8s/blob/f3ed1cef1494f9a90ea111c03b5c2d9e00a5b72c/scripts/installer-for-ibm-storage-enabler-for-containers/ubiquity_uninstall.sh#L21)):*
<details><summary><i>Previously, alonm wrote…</i></summary><blockquote>

issing 'n' in uninstalls
</blockquote></details>

good catch

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ibm/ubiquity-k8s/147)*

**Pull request merged**